File: solved-easy/runningSum.js
var runningSum = function (nums) {
	let result = [nums[0]];
	nums.reduce(function (a, b) {
		result.push(a + b);
		return a + b;
	});
	return result;
};
console.log(runningSum([1, 2, 3, 4]));

File: 75_python/running_sum.py
# https://leetcode.com/problems/running-sum-of-1d-array/
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        result = []
        total = 0
        for i in range (len(nums)):
            total += nums[i]
            result.append(total)

        return result
